docs/getting-started/first-autoscale.md

```diff
@@ -17,7 +17,7 @@ You can watch the pods and see how they ==scale to zero== after traffic stops go
 kubectl get pod -l serving.knative.dev/service=hello -w
 ```
 
-### Expected Output
+**Expected output:**
```

Comment:
```suggestion
==**Expected output:**==
```

Reply:
I will accept but highlighting boring text, I think is not optimal